blackbox-exporter/post_install.md

```diff
@@ -0,0 +1,78 @@
+# Prometheus-Blackbox-Exporter
```

Comment:
Can we have only the essential post install steps here and then point to relevant link. This looks too big to fit